### ISSUE TITLE
tmp files aren't deleted

### ISSUE THREAD

**Issue by toketin** [mafintosh/peerflix#252]
Hi, i've discovered that even if i run "peerflix -k -r "magnet link"", once the video ends and mpv close itself, the downloaded file into /tmp aren't deleted. I'm using Peerflix 0.32 on Archlinux.

**Comment by joshterrill** [comment]
Should they delete themselves? What if you want to watch them later?

**Comment by toketin** [comment]
Yes with -r option it should delete the file once the streaming is ended.

**Comment by KeizerDev** [comment]
Are the files still there after reboot? Peerflix is keeps the tmp till reboot for resuming your video when quit. Check out #260
